code/lib/blocks/src/components/ArgsTable/ArgRow.tsx

```diff
@@ -87,7 +88,7 @@ export const ArgRow: FC<ArgRowProps> = (props) => {
         <Name>{name}</Name>
         {required ? <Required title="Required">*</Required> : null}
       </StyledTd>
-      {compact ? null : (
+      {compact || hideColumns?.includes('descrption') ? null : (
```

Comment:
Nice work! Looks like there's a minor typo here:

```suggestion
      {compact || hideColumns?.includes('description') ? null : (
```

Reply:
good catch. I should probably check my story too. Haha feel like my story wouldn't have been working. Thanks!